### ISSUE TITLE
Contribution tool commit lines of code / files affected

### ISSUE THREAD

**Issue by HGustavs** [HGustavs/LenaSYS#9836]
The current version of the contribution tool does not show the size of commits

I would like to visualize the size of the commits in some manner, for example the size of the circle could be larger if more lines are committed. 

<img width="1030" alt="Screenshot 2021-04-05 at 16 11 04" src="https://user-images.githubusercontent.com/6978801/113583151-90143100-9629-11eb-877b-80540cdf8397.png">

A second related issue is that there is no legend showing what each of the lines represent

**Comment by a19anjst** [comment]
There is a difference between making one line of code in a hundred files and several lines of code in one file
Want to be able to see this difference as several lines of code in one file are worth more than one line of code in several different files

**Comment by a18willi** [comment]
Through investingating how this could be implemented we came to realize that commits does not register in the contribution tool. If commits doesnt register at all or if they register under a separate tag such as "events" is unclear. We know that gitcommits exists in the database as we have found them there but due to some bug these commits doesnt get saved under commits in contribution.php. Therefore this bug needs to be fixed before this task can be implemented.
